deepgram/transcription.py

```diff
@@ -111,6 +189,18 @@ async def _receiver(self) -> None:
 
     def _ping_handlers(self, event_type: LiveTranscriptionEvent,
                        body: Any) -> None:
+        """
+        The _ping_handlers function is a callback that is called when the
+        transcription service sends a ping event.  It calls all of the functions
+        in self.handlers, which are registered by calling add_ping_handler().
+        
+        :param self: Used to Access the attributes and methods of the class.
```

Comment:
Please remove this line from docstring `:param self: Used to Access the attributes and methods of the class.`